src/mesh/p4est_mesh.jl

```diff
@@ -10,9 +10,9 @@ to manage trees and mesh refinement.
 mutable struct P4estMesh{NDIMS, RealT<:Real, NDIMSP2} <: AbstractMesh{NDIMS}
   p4est                 ::Ptr{p4est_t}
   p4est_mesh            ::Ptr{p4est_mesh_t}
-  tree_node_coordinates ::Array{RealT, NDIMSP2} # [dimension, i, j, k, tree_id]
+  tree_node_coordinates ::Array{RealT, NDIMSP2} # [dimension, i, j, k, tree]
```

Comment:
Can you please add a comment regarding what exactly is stored in this array? I believe it's the "DG" nodes that define the geometry of the respective tree (specifically, the level-0 cell of each tree), but I might be mistaken.